libs/shared/storyblok-sdk/src/hooks/useStoryblok.ts

```diff
@@ -11,7 +11,7 @@ interface CustomWindow extends Window {
 declare const window: CustomWindow;
 
 export const Storyblok = new StoryblokClient({
-  accessToken: process.env['NEXT_PUBLIC_STORYBLOK_TOKEN'],
+  accessToken: process.env['STORYBLOK_TOKEN'],
```

Comment:
https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser

This variable should not be accessible without prefix on client side.
So, if we use this StoryblokClient instance to fetch data on client side we won't see an error because it's going to work without token too. Thing is that in this case we can only fetch data that is published an publicliy available.